src/app/modules/main/wallet_section/activity/entry.nim

```diff
@@ -194,6 +182,7 @@ QtObject:
       return newQVariant(newCurrencyAmount())
     return newQVariant(self.transaction[].getTotalFees())
 
+  # TODO: Why is this used in the TransactionDelegate, getDetailsString? Is it needed for displaying the header?
```

Comment:
@Cuteivist, this is only for the details page, right? Can be moved to an entry_details.nim and fetched on request?

Reply:
I will follow up with work in this context as discussed https://github.com/status-im/status-desktop/issues/11597